Workbooks/Citrix.json

```diff
@@ -9,287 +9,1745 @@
+                    "title": "Data Downloaded (in GB)",
+                    "timeContext": {
+                      "durationMs": 2592000000
+                    },
+                    "timeContextFromParameter": "timeRange",
+                    "queryType": 0,
+                    "resourceType": "microsoft.operationalinsights/workspaces",
+                    "visualization": "tiles",
+                    "tileSettings": {
+                      "titleContent": {
+                        "columnMatch": "Column1",
+                        "formatter": 12,
+                        "formatOptions": {
+                          "palette": "blue"
+                        }
+                      },
+                      "showBorder": false
+                    }
+                  },
+                  "customWidth": "50",
+                  "name": "query - 12",
+                  "styleSettings": {
+                    "showBorder": true
+                  }
+                },
+                {
+                  "type": 3,
+                  "content": {
+                    "version": "KqlItem/1.0",
+                    "query": "CitrixAnalytics_userProfile_CL\n| where event_type_s == \"userProfileUsage\"\n| summarize sum(uploaded_bytes_d)/1000000000",
+                    "size": 4,
+                    "title": "Data Uploaded (in GB)",
+                    "timeContext": {
+                      "durationMs": 2592000000
+                    },
+                    "timeContextFromParameter": "timeRange",
+                    "queryType": 0,
+                    "resourceType": "microsoft.operationalinsights/workspaces",
+                    "visualization": "tiles",
+                    "tileSettings": {
+                      "titleContent": {
+                        "columnMatch": "Column1",
+                        "formatter": 12,
+                        "formatOptions": {
+                          "palette": "blue"
+                        }
+                      },
+                      "showBorder": false
+                    }
+                  },
+                  "customWidth": "50",
+                  "name": "query - 13",
+                  "styleSettings": {
+                    "showBorder": true
+                  }
+                }
+              ]
+            },
+            "name": "Content Collaboration Data Usage Details Group"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "selectTab",
+        "comparison": "isEqualTo",
+        "value": "User Profile"
       },
-      "customWidth": "50",
-      "name": "query - 2",
-      "styleSettings": {
-        "maxWidth": "50%",
-        "showBorder": true
-      }
+      "name": "User Profile Group"
     }
-    ], 
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/c466abac-2da4-4d72-8c50-11f292f37e01/resourcegroups/jaystestrg/providers/microsoft.operationalinsights/workspaces/jays-test-workspace"

```

Comment:
@jmuthuku : Workbooks Template Validation Failing because of this. A workbook template must not contain any references to resources. Please take a look and resolve. Thanks

Reply:
Thanks